### PR TITLE
add frame for linux platform

### PR DESCRIPTION
**Pull request by notAlaanor**
| Q                | A
| ---------------- | ---
| Bug fix?         | yes
| Fixed tickets    | #23

### Description

Window is created with a frame on Linux platforms.

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Could the two commits be squashed?

File: src/main/createWindow.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const createWindow = (pathname, options = {}) => {
     },
     useContentSize: true,
     show: false,
-    frame: false,
+    frame: (process.platform === 'linux' ? false : true),
```
Comment (Jocs) [inline src/main/createWindow.js]: frame: process.platform === 'linux',

So don't need to use condition ? expr1 : expr2'

**Comment by notAlaanor** [comment]
Whoops, should be fixed now.

**Comment by SuperSandro2000** [comment]
@notAlaanor could you please drop/squash those comments and git push -f to keep the commit history clear of clutter?

File: src/main/createWindow.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ const createWindow = (pathname, options = {}) => {
     },
     useContentSize: true,
     show: false,
-    frame: false,
+    frame: (process.platform === 'linux'),
```
Comment (Jocs) [inline src/main/createWindow.js]: This PR is helpful to the Linux users, One more thing, I think the `(` and `)` are not needed.

Can you remove them, then I will merge this PR.

**Comment by Jocs** [comment]
@notAlaanor  I will release v0.7.16 after merge this PR.

File: package.json [hunk 3]
```diff
@@ -116,6 +116,7 @@
     "css-loader": "^0.28.4",
     "del": "^3.0.0",
     "devtron": "^1.4.0",
+    "electron": "^2.0.0-beta.4",
```
Comment (Jocs) [inline package.json]: A ha, I just degrade electron to 2.0.0-beta.1, this change maybe conflict with the last commit I just push.

**Pull request merged**